webapp/channels/src/components/sidebar_right/sidebar_right.tsx

```diff
@@ -173,7 +182,7 @@ export default class SidebarRight extends React.PureComponent<Props, State> {
             } else {
                 setTimeout(() => {
                     if (this.previousActiveElement) {
-                        focusElement(this.previousActiveElement, false);
+                        focusElement(this.previousActiveElement, true);
```

Comment:
Why change this to true?

Reply:
oh, it should be true so it only sets the blue border on focus when it is a keyboard event.

Reply:
I think passing `false` would maintain the current state of whether or not the keyboard highlight is visible, but I could be wrong there.

This is definitely another case where I want to get rid of this part of the `A11yController` and replace it with `:focus-visible`